go/libraries/doltcore/sqle/stats/stats_provider.go

```diff
@@ -26,17 +27,20 @@ import (
 
 	"github.com/dolthub/dolt/go/libraries/doltcore/doltdb"
 	"github.com/dolthub/dolt/go/libraries/doltcore/schema"
+	"github.com/dolthub/dolt/go/libraries/doltcore/sqle"
 	"github.com/dolthub/dolt/go/libraries/doltcore/sqle/dsess"
 	"github.com/dolthub/dolt/go/libraries/doltcore/sqle/dtables"
 	"github.com/dolthub/dolt/go/store/hash"
 	"github.com/dolthub/dolt/go/store/prolly"
+	"github.com/dolthub/dolt/go/store/prolly/tree"
 )
 
 var ErrFailedToLoad = errors.New("failed to load statistics")
 
 type DoltStats struct {
 	level         int
 	chunks        []hash.Hash
+	active        map[hash.Hash]int
```

Comment:
Can you add docstrings explaining the purpose of these fields? `active` in particular is unclear to me.